addons/point_of_sale/static/src/app/generic_components/internal_note/internal_note.js

```diff
@@ -0,0 +1,32 @@
+import { Component } from "@odoo/owl";
+
+export class InternalNote extends Component {
```

Comment:
this component is not specific to "notes". It can be called something more generic, like "Tags"